src/main/java/edu/tamu/scholars/middleware/discovery/controller/AbstractSolrDocumentController.java

```diff
@@ -53,6 +54,11 @@
         return ResponseEntity.ok(new Count(repo.count(query, facets, params)));
     }
 
+    @GetMapping(value = "/recently-updated", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
```

Comment:
Make path `/search/recently-updated`. This affords mapping a link to the HAL links of the collection resource. Please add link in `SorDocumentResourcesSearchProcessor`. Also, remove the produces property. The content type is handled by the assembler. Which the content type will be `HAL_JSON_UTF8_VALUE`. Update test accordingly.